readme.md

```diff
@@ -7,6 +7,35 @@ $ npm -g install gimme-stat
 $ cd /home/your_git_project_name
 $ gimme-stat
 ```
+# Usage
```

Comment:
Please rename the section to 'Usage as nodejs module' and relocate the section at the end